lib/runner/extensions/event.command.js

```diff
@@ -410,6 +413,25 @@ module.exports = {
                     // upcoming commands(request, httprequest).
                     result && result.request && (payload.context.request = result.request);
 
+                    visualizerResults = result && result.return && result.return.visualizer;
```

Comment:
extract visualiser results and bubble it via Item callback ... similar to how set next request decision is taken.